src/features/projectsV2/ProjectsMap/InterventionDropDown/index.tsx

```
@@ -0,0 +1,84 @@
+          <InterventionIcon width={27} color={'#333'} />
+          <>
+            {selectedSiteData && (
+              <div className={styles.labelTextContainer}>
+                {isMobile?<label className={styles.sitesLabel}>{truncateString(selectedSiteData?.label, 40)}
+                </label>:
+                <p className={styles.siteName}  style={{marginTop:'5px'}}>
+                  {truncateString(selectedSiteData?.label, 40)}
+                </p>}
+              </div>
+            )}
+          </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)